Objective-C/ChartboostExample/ViewController.m

```diff
@@ -21,9 +21,9 @@ @implementation ViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    self.interstitial = [[CHBInterstitial alloc] initWithLocation:CBLocationDefault delegate:self];
-    self.rewarded = [[CHBRewarded alloc] initWithLocation:CBLocationDefault delegate:self];
-    self.banner = [[CHBBanner alloc] initWithSize:CHBBannerSizeStandard location:CBLocationDefault delegate:self];
+    self.interstitial = [[CHBInterstitial alloc] initWithLocation:@"" delegate:self];
+    self.rewarded = [[CHBRewarded alloc] initWithLocation:@"" delegate:self];
+    self.banner = [[CHBBanner alloc] initWithSize:CHBBannerSizeStandard location:@"" delegate:self];
```

Comment:
since this is an example app we can leave the location as "default" instead of ""

Reply:
ok!

Reply:
same for the swift project 🙌